data/World/Ganons Castle.json

```diff
@@ -38,7 +38,9 @@
             "Forest Trial Clear": "can_use(Light_Arrows) and (Fire_Arrows or Dins_Fire)"
         },
         "locations": {
-            "Ganons Castle Forest Trial Chest": "True"
+            "Ganons Castle Forest Trial Chest": "True",
+            "Ganons Castle Forest Trial Pot 1" : "can_use(Fire_Arrows) or (can_use(Dins_Fire) and can_use(Bow))",
+            "Ganons Castle Forest Trial Pot 2" : "can_use(Fire_Arrows) or (can_use(Dins_Fire) and can_use(Bow))"
```

Comment:
You can Hookshot up next to the torch and Din's Fire it from there. Since Hookshot can be an alternative to Bow.